server/lib/nl/fulfillment/base.py

```diff
@@ -175,6 +177,17 @@ def _add_charts(state: PopulateState, places: List[Place],
   return found
 
 
+def overview_fallback(state: PopulateState, places: List[Place],
+                      chart_origin: ChartOriginType) -> bool:
+  # Overview chart is a safe fallback.
+  state.block_id += 1
+  chart_vars = ChartVars(svs=[],
+                         block_id=state.block_id,
+                         include_percapita=False)
+  return add_chart_to_utterance(ChartType.PLACE_OVERVIEW, state, chart_vars,
```

Comment:
is this the overview for the first place in the list?

Reply:
Yes, good point.  As of now, there is only one place here.  When we handle multiple places from query PLACE_OVERVIEW handling in page_config_next.py might need improvement.